pandas/tests/strings/test_split_partition.py

```diff
@@ -92,81 +103,42 @@ def test_split_object_mixed():
     assert isinstance(result, Series)
     tm.assert_almost_equal(result, exp)
 
-    result = mixed.str.split("_", expand=False)
-    assert isinstance(result, Series)
-    tm.assert_almost_equal(result, exp)
-
 
 @pytest.mark.parametrize("method", ["split", "rsplit"])
-def test_split_n(any_string_dtype, method):
+@pytest.mark.parametrize("n", [None, 0])
+def test_split_n(any_string_dtype, method, n):
     s = Series(["a b", pd.NA, "b c"], dtype=any_string_dtype)
     expected = Series([["a", "b"], pd.NA, ["b", "c"]])
 
-    result = getattr(s.str, method)(" ", n=None)
-    tm.assert_series_equal(result, expected)
-
-    result = getattr(s.str, method)(" ", n=0)
+    result = getattr(s.str, method)(" ", n=n)
     tm.assert_series_equal(result, expected)
 
 
 def test_rsplit(any_string_dtype):
-    values = Series(["a_b_c", "c_d_e", np.nan, "f_g_h"], dtype=any_string_dtype)
-    result = values.str.rsplit("_")
-    exp = Series([["a", "b", "c"], ["c", "d", "e"], np.nan, ["f", "g", "h"]])
-    tm.assert_series_equal(result, exp)
-
-    # more than one char
-    values = Series(["a__b__c", "c__d__e", np.nan, "f__g__h"], dtype=any_string_dtype)
-    result = values.str.rsplit("__")
-    tm.assert_series_equal(result, exp)
-
-    result = values.str.rsplit("__", expand=False)
-    tm.assert_series_equal(result, exp)
-
     # regex split is not supported by rsplit
     values = Series(["a,b_c", "c_d,e", np.nan, "f,g,h"], dtype=any_string_dtype)
     result = values.str.rsplit("[,_]")
     exp = Series([["a,b_c"], ["c_d,e"], np.nan, ["f,g,h"]])
     tm.assert_series_equal(result, exp)
 
+
+def test_rsplit_max_number(any_string_dtype):
     # setting max number of splits, make sure it's from reverse
     values = Series(["a_b_c", "c_d_e", np.nan, "f_g_h"], dtype=any_string_dtype)
     result = values.str.rsplit("_", n=1)
     exp = Series([["a_b", "c"], ["c_d", "e"], np.nan, ["f_g", "h"]])
     tm.assert_series_equal(result, exp)
 
 
-def test_rsplit_object_mixed():
```

Comment:
where did these go>?

Reply:
Parameterized now in `test_split_object_mixed`